src/Tests/AggregatorCronTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\ultimate_cron\Tests\AggregatorCronTest.
+ */
+
+namespace Drupal\ultimate_cron\Tests;
+
+use Drupal\aggregator\Tests\AggregatorTestBase;
+
+/**
+ * Update feeds on cron.
+ *
+ * @group ultimate_cron
+ */
+class AggregatorCronTest extends AggregatorTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    \Drupal::service('module_installer')->install(array('ultimate_cron'));
+  }
+
+  /**
+   * Adds feeds and updates them via cron process.
+   */
+  public function testCron() {
+    // Create feed and test basic updating on cron.
+    $this->createSampleNodes();
+    $feed = $this->createFeed();
+    $this->cronRun();
+    $this->assertEqual(5, db_query('SELECT COUNT(*) FROM {aggregator_item} WHERE fid = :fid', array(':fid' => $feed->id()))->fetchField());
+    $this->deleteFeedItems($feed);
```

Comment:
I think we really only need it up until here. The remaining things are testing aggregator.module, not ultimate_cron.